src/installer/pkg/sfx/Microsoft.NETCore.App/monocrossaot.sfxproj

```diff
@@ -0,0 +1,27 @@
+<Project Sdk="Microsoft.Build.Traversal" DefaultTargets="BuildMonoCrossAllTargets">
```

Comment:
If we’re using the traversal SDK, then we should hook into the Traversal targets instead of defining our own.